### DISCUSSION TITLE
Access privately forwarded ports from alternative clients such as Postman

### DISCUSSION THREAD

**Discussion by tom-elvidge** [community/community#19555] [Codespaces] [answered]
As far as I am aware the only way to access privately forwarded ports is from your browser which has the authentication cookie described [here](https://docs.github.com/en/enterprise-cloud@latest/codespaces/codespaces-reference/security-in-codespaces#port-forwarding) or from the browser built into the Visual Studio Code web app. This works great when building web user interfaces, but I want to use Postman to make debugging requests to a REST service being developed in Codespaces, and Postman is not authenticated to GitHub, so it cannot access my privately forwarded port.

I don't want to use publicly forwarded ports and I don't want to use desktop Visual Studio Code (where I believe I could bind to localhost). Does anyone know if I can access privately forwarded ports from Codespaces in the web to alternative clients, and how I would configure that?

**Comment by aishwaryabh** [comment, marked as answer]
Yes this is a feature that we support; you can use Postman to forward ports! You can obtain the GitHub token by typing `echo $GITHUB_TOKEN` in the terminal and use that value for the custom header `X-GitHub-Token`. Let us know if you have any questions!

**Reply by tom-elvidge** [reply]
Perfect, thank you!
